scripts/create-hosted-graphite-dashboards.py

```diff
@@ -0,0 +1,35 @@
+#!/usr/bin/env python
+# -*- coding: utf-8 -*-
+"""
+Description:
+    To add
+
+Usage:
+    scripts/create-hosted-graphite-dashboards.py <hosted_graphite_api_key>
+
+Example:
+    scripts/create-hosted-graphite-dashboards.py apikey
+"""
+
+import os
+import sys
+
+import requests
+from docopt import docopt
+
+sys.path.insert(0, '.')  # noqa
+
+
+def generate_dashboards(api_key):
+    endpoint = "https://api.hostedgraphite.com/api/v2/grafana/dashboards/"
+    path = "scripts/templates/grafana-dashboards/"
```

Comment:
I'd maybe think about moving the templates to a top-level directory alongside kibana (like `graphite` or `grafana`) and adding it to the README as it took me a bit to find it in the checkout.